src/main/java/org/checkerframework/specimin/JavaParserUtil.java

```diff
@@ -373,4 +360,101 @@ static String removeMethodReturnTypeAndAnnotationsImpl(String declAsString) {
     String result = methodWithoutReturnType.replace("< ", "<");
     return result;
   }
+
+  /**
+   * Checks to see if an expression (usually a field or method) is static.
+   *
+   * @param expr The expression
+   * @return Whether it is static or not
+   */
+  public static boolean isAStaticMember(Expression expr) {
+    CompilationUnit cu = expr.findCompilationUnit().get();
+
+    String nameOfExpr;
+    String nameOfScope = null;
+    Expression scope = null;
+
+    if (expr.isNameExpr()) {
+      nameOfScope = nameOfExpr = expr.asNameExpr().getNameAsString();
```

Comment:
This double-assignment pattern is a bit unintuitive to read and is usually considered bad style in Java. It's more verbose to use three lines for this, but I'd prefer it.